src/constant/api.constant.js

```diff
@@ -1,5 +1,6 @@
-const ApiConstant = {
+export const ApiConstant = {
   auth: {
-    login: 'auth/login'
-  }
+    login: 'auth/login',
+  },
+  product: 'api/v1/product'
```

Comment:
mình config base url đã có api/v1 rồi, chỗ này chỉ cần 'product'
```suggestion
  product: 'product'
```

Reply:
Sorry đoạn này mình chưa config vào base url mà mình để api/v1 luôn ở .env
Bạn thêm `api/v1` vào `baseURL` của 2 instance ở [src/api/index.js](https://github.com/phuonghight/HIT-FOOD-Client-Vue/blob/master/src/api/index.js) giúp mình nhé.

Reply:
ok